sdk/js/src/Client.ts

```diff
@@ -167,7 +170,10 @@ export class DevCycleClient<
         }
         this.initializeTriggered = true
 
-        const storedAnonymousId = await this.store.loadAnonUserId()
+        // don't wait to load anon id if we're being provided with a real one
```

Comment:
this is to fix a problem where initializing a bootstrapped client couldn't be used for variable evaluation until the next tick, because the config wasn't set until after this call